dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -54,7 +54,7 @@
   # Hopefully fixes many of the issues with font wiggle due to lazily loading
   # alternative fonts for symbols and localized glyphs.
   wait_until do
-    @browser.execute_script('return !!(await document.fonts.ready)') == true
+    @browser.execute_script('return document.fonts.status === "loaded"') == true
```

Comment:
the `wait_until` function is actually polling, not handling async requests at all. but the [FontFaceSet](https://developer.mozilla.org/en-US/docs/Web/API/FontFaceSet) api has a `status` property we can poll for.